lib/KTable/KTableGridItem.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+
+  <td :style="{ textAlign: align }" tabindex="0" @keydown="onKeydown">
+    {{ content }}
+  </td>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'KTableGridItem',
+    props: {
+      content: {
```

Comment:
In the next iteration, I'd recommend using slot rather than `content` prop, since we will need to work with rich HTML content. This will make transitioning towards exposing scoped slots easier.